DuckDuckGo/TabViewController.swift

```diff
@@ -88,6 +88,10 @@ class TabViewController: UIViewController {
     internal lazy var featureFlagger = AppDependencyProvider.shared.featureFlagger
     private lazy var featureFlaggerInternalUserDecider = AppDependencyProvider.shared.featureFlaggerInternalUserDecider
 
+    private lazy var autofillUrlMatcher: AutofillDomainNameUrlMatcher = AutofillDomainNameUrlMatcher()
```

Comment:
Is this reference to autofillUrlMatcher needed, or could it be initialised in the call to the AutofillWebsiteAccountMatcher init?

Reply:
Agree, not needed - have made that change